### DISCUSSION TITLE
GPTH doesn't actually copy my photos

### DISCUSSION THREAD

**Discussion by ylanraptor** [TheLastGimbus/GooglePhotosTakeoutHelper#226] [I NEED HELP]
I followed all the steps on windows correctly, but the process gets stuck at "Finding albums", then exits and my new folder is left containing a single folder "ALL PHOTOS" and inside "2015", an empty folder.
What's the issue?

**Comment by TheLastGimbus** [comment]
> process gets stuck at "Finding albums", then exits

really? like, crashes, or just cloeses, or what? can you show me screenshot of what happens?

if it just suddently closes, maybe try to watch system monitor/task manager to see if it's not overflowing RAM or something
